WalletWasabi/BitcoinCore/Monitoring/RpcFeeProvider.cs

```diff
@@ -40,6 +40,11 @@ protected override async Task ActionAsync(CancellationToken cancel)
 			}
 			InError = false;
 		}
+		catch(NoEstimationException ex)
+		{
+			Logging.Logger.LogInfo("Couldn't get fee estimation from Bitcoin Core, probably because it was not yet initialized.");
```

Comment:
Nit: Given that Wasabi now ships with Bitcoin Knots should we mention that instead of Bitcoin Core or should say "Bitcoin node" or something?